electron/package.json

```diff
@@ -7,9 +7,11 @@
   "devDependencies": {
     "@types/node": "12.19.16",
     "@wireapp/copy-config": "1.1.13",
+    "adm-zip": "0.5.2",
```

Comment:
Why not your own: https://github.com/ffflorian/jszip-cli :)

Reply:
Not the right use case. `jszip-cli` is supposed to be used as CLI.